src/app/pages/Sandbox/Editor/Workspace/EntryContainer.js

```diff
@@ -6,7 +6,7 @@ export const getContainerStyles = props => {
     ? theme.primary
     : theme.templateColor || theme.secondary;
   let styles = `
-    ${props => props.noTransition || 'transition: 0.3s ease all;'}
+    ${props.noTransition || 'transition: 0.3s ease all;'}
```

Comment:
I think we should keep this the same (maybe rename `props`), because the passed in props are different from the component props in the case of styled-components.

Reply:
I didn't know that. There are a few more, I'll refactor them. `styleProps` is ok?

Reply:
Hmm, yes, sounds good!